src/archiefvernietigingscomponent/destruction/filters.py

```diff
@@ -34,3 +35,40 @@ def filter_reviewed(self, queryset, name, value):
             return queryset.reviewed_by(self.request.user)
         else:
             return queryset
+
+
+class RecordManagerListFilter(FilterSet):
+    list_status = ChoiceFilter(
+        field_name="list_status",
+        label="lists for record manager",
+        choices=RecordManagerDisplay.choices,
+        method="filter_list_status",
+        empty_label=None,
+        widget=forms.RadioSelect,
+    )
+
+    class Meta:
+        model = DestructionList
+        fields = ("list_status",)
+
+    def __init__(self, data, *args, **kwargs):
+        data = data or {}
+        if not data.get("list_status"):
+            data["list_status"] = RecordManagerDisplay.all
+
+        super().__init__(data, *args, **kwargs)
+
+    def filter_list_status(self, queryset, name, value):
+        if value == RecordManagerDisplay.in_progress:
+            return queryset.filter(
+                Q(status=ListStatus.in_progress) | Q(status=ListStatus.processing),
+                ~Q(author=F("assignee")) | Q(assignee__isnull=True),
```

Comment:
```suggestion
                Q(~Q(author=F("assignee"), status=ListStatus.in_progress) | Q(status=ListStatus.processing),
```
I'm not sure but I thought that `Q(assignee__isnull=True)` is a special case of `~Q(author=F("assignee"))` and is excessive
And I know that theoretically, an author can't be an assignee if `status=ListStatus.processing`, but still this case should be included in this `if`